pyvista/__init__.py

```diff
@@ -9,23 +9,272 @@
 # Load default theme.  Must be loaded first.
 from pyvista._version import __version__
 from pyvista.plotting import *
-from pyvista.utilities import *
-from pyvista.core import *
+from pyvista.utilities import (
+    # arrays,
+    cells,
+    # common,
+    # errors,
+    # features,
+    fileio,
+    # geometric_objects,
+    helpers,
+    # misc,
+    # parametric_objects,
+    reader,
+    # regression,
+    # sphinx_gallery,
+    transformations,
```

Comment:
This is here for `pyvista.transformations.apply_transformation_to_points` (in docs) and `pyvista.transformations.axis_angle_rotation` (in a handful of tests), and neither names are available at the top-level namespace.